src/D2L.CodeStyle.Analyzers/Language/OnlyVisibleToAnalyzer.Model.cs

```diff
@@ -25,19 +25,20 @@ INamedTypeSymbol onlyVisibleToTypeAttribute
 			}
 
 			public static Model? TryCreate( Compilation compilation ) {
+				INamedTypeSymbol? onlyVisibleToTypeAttribute = compilation
+					.GetTypeByMetadataName( OnlyVisibleToTypeAttributeMetadataName );
 
-				INamedTypeSymbol? onlyVisibleToTypeAttribute = compilation.GetTypeByMetadataName( OnlyVisibleToTypeAttributeMetadataName );

```

Comment:
Aside (since I'm not looking at the implementation right now), extra non-changes like this make things a lot harder to review.